src/ConnectedMode/UI/ManageConnections/ManageConnectionsDialog.xaml

```diff
@@ -1,120 +1,138 @@
+            <wpf:BoolToVisibilityConverter x:Key="TrueToVisibleConverter" FalseValue="Collapsed" TrueValue="Visible" />
         </ResourceDictionary>
     </Window.Resources>
     <Window.Style>
-        <Style TargetType="Window" BasedOn="{StaticResource ConnectedModeWindow}"></Style>
+        <Style TargetType="Window" BasedOn="{StaticResource ConnectedModeWindow}" />
     </Window.Style>
 
     <Grid DataContext="{Binding ElementName=This, Path=ViewModel}" Margin="10">
         <Grid.RowDefinitions>
-            <RowDefinition Height="*"/>
-            <RowDefinition Height="Auto"/>
-            <RowDefinition Height="Auto"/>
+            <RowDefinition Height="*" />
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="Auto" />
         </Grid.RowDefinitions>
 
         <Grid Grid.Row="0">
-           <Grid.RowDefinitions>
-               <RowDefinition Height="Auto"/>
-               <RowDefinition Height="*"/>
-           </Grid.RowDefinitions>
-           <Grid.ColumnDefinitions>
-               <ColumnDefinition Width="*"/>
-           </Grid.ColumnDefinitions>
+            <Grid.RowDefinitions>
+                <RowDefinition Height="Auto" />
+                <RowDefinition Height="*" />
+            </Grid.RowDefinitions>
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="*" />
+            </Grid.ColumnDefinitions>
 
             <Grid Grid.Row="0">
                 <Grid.ColumnDefinitions>
-                    <ColumnDefinition Width="*"/>
-                    <ColumnDefinition Width="Auto"/>
+                    <ColumnDefinition Width="*" />
+                    <ColumnDefinition Width="Auto" />
                 </Grid.ColumnDefinitions>
 
-                <TextBlock Grid.Column="0" Text="{x:Static res:UiResources.ExistingConnectionsLabel}" Style="{StaticResource HeaderTextBlock}"/>
-                <Button Grid.Column="1" Content="{x:Static res:UiResources.NewConnectionButton}" HorizontalAlignment="Right"
-                        Click="NewConnection_Clicked"/>
+                <TextBlock Grid.Column="0" Text="{x:Static res:UiResources.ExistingConnectionsLabel}"
+                           Style="{StaticResource HeaderTextBlock}" />
+                <Button Grid.Column="1" Content="{x:Static res:UiResources.NewConnectionButton}"
+                        HorizontalAlignment="Right"
+                        Click="NewConnection_Clicked" />
             </Grid>
-           <ListBox Grid.Row="1" ItemsSource="{Binding Path=ConnectionViewModels, Mode=OneWay}" Margin="0,10" 
-                    ItemContainerStyle="{StaticResource NoSelectionListBoxItemStyle}">
-               <ListBox.ItemTemplate>
-                   <DataTemplate>
+            <ListBox Grid.Row="1" ItemsSource="{Binding Path=ConnectionViewModels, Mode=OneWay}" Margin="0,10"
+                     ItemContainerStyle="{StaticResource NoSelectionListBoxItemStyle}">
+                <ListBox.ItemTemplate>
```

Comment:
It looks like there isn't enough padding/margin. It makes the text difficult to read. Would it be possible to add back some spacing between the lines and between the icon and text?

Keep in mind that on the second line there is hidden but still there the smart notifications checkbox

![image](https://github.com/user-attachments/assets/3bb71ba9-00a1-426a-8077-5131def0ac84)